### PR TITLE
Admin Ranks / Config

### PR DESCRIPTION
**Pull request by JohnOxford**
https://docs.google.com/document/d/1s6W2X7DAJoFpwmXjnR-SQfDiiJp--OOfPrVupg_p32s/edit?usp=sharing

document explaining some stuff about structure and purpose and what not

PR changes the station name to Mojave wasteland, since we are based in Nevada according to the MOTD, 

it changes the github link to the correct one, and sets all the ranks in accordance with the document (sort of)

### CODE REVIEW

**Comment by UndeadNappist** [comment]
Need to hang onto Three Dog and Host role, use +@ and rearrange the stack, would like to hang onto moderator or admin candidate roles (which should be used more often let's be honest).

**Comment by UndeadNappist** [comment]
Also you need to add your additional, new, shiny ranks IN ADDITION to the old ranks, move everyone over to the new ranks, THEN delete the old ranks.

**Comment by Raze500** [comment]
Question, ¿ this will modify/add roles to the newest staff on the server?

**Comment by JohnOxford** [comment]
host is high elder, +@ isn't entirely necessary, repeating the same roles from the previous ones allows you explicitly see which permissions each role has. i'll fix hang on

**Comment by UndeadNappist** [comment]
I realize host is high elder, but if I'm not mistaken it plugs into a lot of other configs and if you remove host things will break.

So don't do that.

Three Dog role is literally our version of the DJ role, and is already fallout themed.

**Comment by JohnOxford** [comment]
understandable, after the ranks are changed we'll remove the old ones, when that happens i'll just organize the list and include three dog as one of the ranks that transfered

three dog doesn't appear in new vegas, objectively bad admin rank 👎

**Comment by JohnOxford** [comment]
aspirant is also representitive of the admin candidate/trial admin/moderator role

**Comment by companoo** [comment]
admin ranks should have varedit

**Comment by JohnOxford** [comment]
does that warrant coder ranks getting +admin then?

i dont want to be onesided about it

**Comment by companoo** [comment]
coder doesent need +admin, admin ranks need to be able to varedit

**Comment by UndeadNappist** [comment]
> admin ranks should have varedit

Trialmins and fullmins screw up with mass varedit and a bunch of the more dangerous server-wide controls all the time. It's kind of aggravating.

**Comment by JohnOxford** [comment]
paladins and sentinels are intelligent enough to comprehend varedit and not do something retarded, it goes under the same context that you wouldnt want scribes to have admin powers out of fear they'd do something retarded

**Comment by JohnOxford** [comment]
but yes, knights and aspirants don't need varedit

**Comment by UndeadNappist** [comment]
> paladins and sentinels are intelligent enough to comprehend varedit and not do something retarded, it goes under the same context that you wouldnt want scribes to have admin powers out of fear they'd do something retarded

They've accidentally muted server-wide speech three times as far as I know.

**Comment by improvedname** [comment]
just tie mass edits to +debug 💤

**Comment by companoo** [comment]
looks good to me, i agree with knights + aspirants not getting varedit

**Comment by improvedname** [comment]
The joke that VV is needed in basic admin stuff, and the common trope is don't mess with things that you don't know how they work.

File: config/admin_ranks.txt [hunk 1]
```diff
@@ -47,3 +47,13 @@ Host			= +EVERYTHING *EVERYTHING
...
 Head Coder      = +EVERYTHING
 Coder			= +DEBUG +VAREDIT +SERVER +SPAWN +POLL -AUTOLOGIN
+
+
+Aspirant 	= +ADMIN +BAN +REJUV
+Knight 		= +ADMIN +BAN +REJUV +STEALTH +FUN +POSSESS +SOUND +SPAWN
+Paladin		= +ADMIN +BAN +REJUV +STEALTH +FUN +POSSESS +BUILD +SOUND +SPAWN +SERVER +RIGHTS +DJ +POLL +VAREDIT
+Sentinel	= +ADMIN +BAN +REJUV +STEALTH +FUN +POSSESS +BUILD +SOUND +SPAWN +SERVER +RIGHTS +DJ +POLL +VAREDIT
+Scribe 		= +STEALTH +POSSESS +REJUV +BUILD +DEBUG +VAREDIT +SOUND +SPAWN +DJ +POLL
+Head Scribe = +FUN +STEALTH +POSSESS +REJUV +BUILD +SERVER +DEBUG +VAREDIT +SOUND +SPAWN +RIGHTS +DJ +POLL
```
Comment (improvedname) [inline config/admin_ranks.txt]: They don't need +Rights, or +Fun

Comment (improvedname) [reply config/admin_ranks.txt]: Neither do they need +DJ or +Sound

File: config/admin_ranks.txt [hunk 2]
```diff
@@ -47,3 +47,13 @@ Host			= +EVERYTHING *EVERYTHING
 
 Head Coder      = +EVERYTHING
 Coder			= +DEBUG +VAREDIT +SERVER +SPAWN +POLL -AUTOLOGIN
+
+
+Aspirant 	= +ADMIN +BAN +REJUV
+Knight 		= +ADMIN +BAN +REJUV +STEALTH +FUN +POSSESS +SOUND +SPAWN
+Paladin		= +ADMIN +BAN +REJUV +STEALTH +FUN +POSSESS +BUILD +SOUND +SPAWN +SERVER +RIGHTS +DJ +POLL +VAREDIT
+Sentinel	= +ADMIN +BAN +REJUV +STEALTH +FUN +POSSESS +BUILD +SOUND +SPAWN +SERVER +RIGHTS +DJ +POLL +VAREDIT
+Scribe 		= +STEALTH +POSSESS +REJUV +BUILD +DEBUG +VAREDIT +SOUND +SPAWN +DJ +POLL
```
Comment (improvedname) [inline config/admin_ranks.txt]: Same as head scribe

File: config/admin_ranks.txt [hunk 3]
```diff
@@ -47,3 +47,13 @@ Host			= +EVERYTHING *EVERYTHING
 
 Head Coder      = +EVERYTHING
 Coder			= +DEBUG +VAREDIT +SERVER +SPAWN +POLL -AUTOLOGIN
+
+
+Aspirant 	= +ADMIN +BAN +REJUV
+Knight 		= +ADMIN +BAN +REJUV +STEALTH +FUN +POSSESS +SOUND +SPAWN
```
Comment (improvedname) [inline config/admin_ranks.txt]: These need +VV

File: config/admin_ranks.txt [hunk 4]
```diff
@@ -47,3 +47,13 @@ Host			= +EVERYTHING *EVERYTHING
 
 Head Coder      = +EVERYTHING
 Coder			= +DEBUG +VAREDIT +SERVER +SPAWN +POLL -AUTOLOGIN
+
+
+Aspirant 	= +ADMIN +BAN +REJUV
```
Comment (improvedname) [inline config/admin_ranks.txt]: Why does this even exist

File: config/admin_ranks.txt (hunk 1 above)
Comment (JohnOxford) [reply config/admin_ranks.txt]: sentinels (head admininstrator) needs to be able to promote in game. paladins double as admin trainers and need to be able to promote people to aspirant.

File: config/admin_ranks.txt (hunk 3 above)
Comment (JohnOxford) [reply config/admin_ranks.txt]: no, the admin candidate rank does not need varedit. compano stated that knights shouldn't get VV under the clause they're likely to fuck something up after we stated paladins are very capable of doing the same thing anyways

File: config/admin_ranks.txt (hunk 4 above)
Comment (JohnOxford) [reply config/admin_ranks.txt]: see above

**Comment by JohnOxford** [comment]
well, we all know if a knight/aspirant fucks up something major with varedit, the most they're going to get is "don't do that again"

so in order to prevent rounds from being ruined in the future, we just wont give it to them,

**Comment by UndeadNappist** [comment]
Need to leave HOST in.

**Comment by RenGusta** [comment]
Change Sentinel to Senior Paladin.

Remove + rights from Paladin

**Pull request merged**